q2_deblur/plugin_setup.py

```diff
@@ -128,8 +122,37 @@ def denoise(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
         'jobs_to_start': qiime2.plugin.Int,
         'hashed_feature_ids': qiime2.plugin.Bool
     },
+    parameter_descriptions={
+        'pos_ref_fp': ("Positive (16S) filtering database. Keep all sequences "
```

Comment:
Drop *(16S)*, since (I don't think) there is anything 16S-specific here.

I think you should also drop `_fp` from this parameter at the others that include it. In some interfaces, the users won't directly be providing a file path so this seems a little CLI specific. 

Also, I would change *this FASTA file* to *these sequences.*, since the user is providing an artifact, not a fasta file.